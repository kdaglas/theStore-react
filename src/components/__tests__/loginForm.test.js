import React from 'react';
import { shallow } from 'enzyme';
import Login from '../loginForm';


describe('login form component', () => {
  it('should return the form without crushing', () => {
    const component = shallow(<Login />);
    expect(component).toMatchSnapshot();
  });
});