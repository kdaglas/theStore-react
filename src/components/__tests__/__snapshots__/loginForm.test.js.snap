// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login form component should return the form without crushing 1`] = `ShallowWrapper {}`;
